admin/test/features/breadcrumbs.feature

```diff
@@ -14,7 +14,8 @@ Feature:
   Scenario: Verify breadcrumb on add pupil page
     Given I am logged in
     And I am on the add pupil page
-    Then I should see the breadcrumb as "Home > Add single pupil"
+#    Then I should see the breadcrumb as "Home > Add single pupil"
```

Comment:
@activemq  Should this commented-out line be removed now?

Reply:
@jon-shipley agreed, I didn't put it in but have removed 👍